packages/lesswrong/components/common/ForumIcon.tsx

```diff
@@ -520,7 +520,8 @@ const ICONS: ForumOptions<Record<ForumIconName, IconComponent>> = {
 
 export type IconProps = {
   className: string,
-  onClick: MouseEventHandler<SVGElement>,
+  onClick?: MouseEventHandler<SVGElement>,
```

Comment:
This is wrapped in `Partial` when it's used so the `?`s aren't needed. For consistency, we should probably either remove the `Partial` and give `className` a `?` as well, or remove all the `?`s as it was before. It's a bit weird having half-and-half.